pages/my-library/style.css

```diff
@@ -170,7 +170,7 @@
 
 .main_container .popular_song .pop_song {
     width: 100%;
```

Comment:
```suggestion
    width: 140px;
    min-width: 140px;
```

Reply:
Set the width as a constant value as well so that the song cards have a constant aspect ratio.
Otherwise the card tend to shrink and become thin.

Reply:
I have added min width 140px so it should not make an issue right?

Reply:
I checked, with only min width the elements in the bottom section change their size and become thinner but adding width seems to fix that.